dataprep/eda/missing/compute/bivariate.py

```diff
@@ -0,0 +1,146 @@
+"""This module implements the plot_missing(df) function's
+calculating intermediate part."""
+
+from typing import List, Optional, Generator, Any
+
+import dask.dataframe as dd
+import numpy as np
+import pandas as pd
+from scipy.stats import rv_histogram
+
+from ...data_array import DataArray
+from ...dtypes import (
+    Continuous,
+    DTypeDef,
+    detect_dtype,
+    is_dtype,
+)
+from ...intermediate import ColumnsMetadata, Intermediate
+from ...staged import staged
+from .common import LABELS, histogram
+
+
+@staged
+def compute_missing_bivariate(  # pylint: disable=too-many-locals
+    df: DataArray,
+    x: str,
+    y: str,
+    bins: int,
+    ndist_sample: int,
+    dtype: Optional[DTypeDef] = None,
+) -> Generator[Any, Any, Intermediate]:
+    # pylint: disable=too-many-arguments
+    """Calculate the distribution change on another column y when
+    the missing values in x is dropped."""
+    df.compute("nulls")
+
+    xloc = df.columns.get_loc(x)
+    yloc = df.columns.get_loc(y)
+
+    col0 = df.values[~df.nulls[:, yloc], yloc].astype(df.dtypes[y])
+    col1 = df.values[~(df.nulls[:, xloc] | df.nulls[:, yloc]), yloc].astype(
+        df.dtypes[y]
+    )
+
+    minimum, maximum = col0.min(), col0.max()
+
+    hists = [
+        histogram(col, dtype=dtype, bins=bins, return_edges=True)
+        for col in [col0, col1]
+    ]
+
+    quantiles = None
+    if is_dtype(detect_dtype(df.frame[y], dtype), Continuous()):
+        quantiles = [
```

Comment:
From the DataArray docstring "the cost for getting the array from a dask DataFrame is non trivial"; so I wonder if it would be better not to call `from_dask_array` repeatedly, and I also think it could be better to pass the list directly into `quantile` rather than iterate over it

Reply:
Correct